provider/cmd/pulumi-resource-cloudinit/main.go

```diff
@@ -17,17 +17,20 @@
 package main
 
 import (
+	"context"
 	_ "embed"
-
 	cloudinit "github.com/pulumi/pulumi-cloudinit/provider"
-	"github.com/pulumi/pulumi-cloudinit/provider/pkg/version"
-	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	//"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/pf/tfbridge"
```

Comment:
If you made this change to `pulumi-resource-cloudinit/main.go`, you need to make an equivalent change to https://github.com/pulumi/pulumi-cloudinit/blob/5feceebee8c4cc719add05382d22bcfd1323f284/provider/cmd/pulumi-tfgen-cloudinit/main.go#L18.